tests/ixia/test_pfc_pause_lossless.py

```diff
@@ -0,0 +1,205 @@
+    vlan_ip_addrs = get_addrs_in_subnet(vlan_subnet, 2)
+       
+    topo_receiver = create_topology(session=session,
+                                    name="Receiver", 
+                                    ports=list(rx_port),
+                                    ip_start=vlan_ip_addrs[0],
+                                    ip_incr_step='0.0.0.1',
+                                    gw_start=gw_addr, 
+                                    gw_incr_step='0.0.0.0')
+
+    topo_sender = create_topology(session=session, 
+                                  name="Sender", 
+                                  ports=list(tx_port), 
+                                  ip_start=vlan_ip_addrs[1], 
+                                  ip_incr_step='0.0.0.1', 
+                                  gw_start=gw_addr, 
+                                  gw_incr_step='0.0.0.0')
+
+    start_protocols(session)
+    
+    test_traffic = create_ipv4_traffic(session=session,
+                                       name='Test Data Traffic',
+                                       source=topo_sender,
+                                       destination=topo_receiver,
+                                       pkt_size=DATA_PKT_SIZE,
+                                       duration=exp_dur,
+                                       rate_percent=50,
+                                       start_delay=1,
+                                       dscp_list=test_dscp_list,
+                                       lossless_prio_list=test_prio_list)
+
+    background_traffic = create_ipv4_traffic(session=session,
+                                             name='Background Data Traffic',
+                                             source=topo_sender,
+                                             destination=topo_receiver,
+                                             pkt_size=DATA_PKT_SIZE,
+                                             duration=exp_dur,
+                                             rate_percent=50,
+                                             start_delay=1,
+                                             dscp_list=bg_dscp_list,
+                                             lossless_prio_list=None)
+    
+    # Pause time duration (in second) for each PFC pause frame.
+    pause_dur_per_pkt = 65535 * 64 * 8.0 / (port_bw * 1000000) 
+    
+    # Do not specify duration here as we want it keep running.
+    pfc_traffic = create_pause_traffic(session=session,
+                                       name='PFC Pause Storm',
+                                       source=rx_port,
+                                       pkt_per_sec=1.1/pause_dur_per_pkt,
+                                       start_delay=0,
+                                       global_pause=False,
+                                       pause_prio_list=test_prio_list)
+    
+    start_traffic(session)
+    
+    # Wait for test and background traffic to finish.
+    time.sleep(exp_dur + 1.5)
```

Comment:
Here, the extra 1.5 seconds is because of the start delay is 1 second (line 94 and 105). We should declare a variable called 'start_delay' and avoid hardcode.

Reply:
Implemented.